program_info/win_install.nsi.in

```diff
@@ -2,6 +2,8 @@
 !include "LogicLib.nsh"
 !include "MUI2.nsh"
 
+!include "x64.nsh"
```

Comment:
Is this required on ARM?

Reply:
yep, it's where the arm check function comes from
https://github.com/kichik/nsis/blob/master/Include/x64.nsh